### ISSUE TITLE
Add `ignoreAtRules: []` to `no-invalid-position-declaration`

### ISSUE THREAD

**Issue by jrmlt** [stylelint/stylelint#8776]
### What minimal example or steps are needed to reproduce the bug?

```scss
@mixin my-mixin() {
  @media (min-width: 900px) {
    font-size: 12px;
  }
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "extends": [
    "stylelint-config-recommended-scss"
  ]
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIACBbAlgDzQOwAQoE8BadLbACgEpdgAdPXVGKNAQ13PWyIHc0oALgAt4uAJwAGCQAcM1Og1y4AZhGwCiAZzQAvGKICMAJlkBuekoC+9SyAA0IZWgA2MAHKsUcRDAyfprgB0YJqa9uBqTgDmCCAKSrQgvgIw2FCaiaIA2hZKuImaAgSuzjgakNjRRABOMJAoXmnMWiEZILkAujb04RXRAGIQ1SisArEAVppq4bDSYYjx+SCFxTCl6plLzqMwhYl2uQVFJWVEfWhRNXUQDamwUC2hm4nbKXvt2LaWQA)

### Which Stylelint-related dependencies are you using?

```json
{
  "stylelint": "latest",
  "stylelint-config-recommended-scss": "latest"
}
```

### What did you expect to happen?

No issue when using at-rule inside mixins.

### What actually happened?

```
3:5-21 error Unexpected invalid position declaration no-invalid-position-declaration
```

### Do you have a proposal to fix the bug?

Quite similar to https://github.com/stylelint/stylelint/issues/8728. The fix could be the same. To add the secondary option `ignoreAtRules: []` as this is a langage extension specific issue.

**Comment by jeddy3** [comment]
@jrmlt Thanks for the request and for using the template.

> Quite similar to https://github.com/stylelint/stylelint/issues/8728. The fix could be the same. To add the secondary option `ignoreAtRules: []` as this is a langage extension specific issue.

SGTM. I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to add a new option](https://stylelint.io/developer-guide/rules#add-an-option-to-a-rule) in the Developer guide.

As you pointed out, the changes in the https://github.com/stylelint/stylelint/pull/8743 PR are a good blueprint.
